src/web/mjs/engine/DiscordPresence.mjs

```diff
@@ -28,8 +28,6 @@ export default class DiscordPresence {
         document.addEventListener( EventListener.onSelectConnector, this._onSelectConnector.bind(this) );
         document.addEventListener( EventListener.onSelectManga, this._onSelectManga.bind(this) );
         document.addEventListener( EventListener.onSelectChapter, this._onSelectChapter.bind(this) );
-
-        this.startDiscordPresence();
```

Comment:
I see, it's redundant due to `this._settings.addEventListener('loaded', this._onSettingsChanged.bind(this));`